core/services/remote_access_services.py

```diff
@@ -121,9 +121,11 @@ def store_trained_classifier_model(job_result_dict):
     if 'classifier_data' not in job_result_dict:
         raise Exception('job_result_dict must contain \'classifier_data\'.')
 
-    payload = job_result_dict
-    payload['vm_id'] = _get_vm_id()
-    signature = generate_signature(payload)
+    payload = {
+        'message': job_result_dict,
+        'vm_id': _get_vm_id()
+    }
+    signature = generate_signature(payload['message'])
     payload['signature'] = signature
```

Comment:
@anmolshkl I don't have opinion on this but I don't see any need to make it explicit. We have already described the structure and what is what on wiki page, so it is not really necessary for the key to be explicit about it.